RepositoryOverview.asciidoc

```diff
@@ -199,7 +199,7 @@ More information about the maven conventions can be found http://maven.apache.or
 
 Finally, each module can have documentation.
 This documentation is located in the `src/docs` folder,
-which is organized as described http://neo4j.com/docs/milestone/community-docs.html#_file_structure_in_emphasis_docs_jar_emphasis[here].
+which is organized as described http://neo4j.com/docs/milestone/resources.html[here].
```

Comment:
Since the description of the `src/docs` folder has been removed, and there isn't really any good replacement for it, I suggest we just cut this sentence and remove the link altogether:

Finally, each module may have documentation, which is located in the `src/docs` folder.